src/describe.rs

```diff
@@ -75,7 +76,7 @@ simple! {
     f64 => F64
     bool => BOOLEAN
     char => CHAR
-    str => STRING
+    str => if cfg!(feature = "enable-interning") { CACHED_STRING } else { STRING }
```

Comment:
I believe this is where the CI issues are coming from. In debug mode this is literally emitting an if/else block and some other assorted instructions, but our "poor man's interpreter" doesn't actually implement if/else/etc. I think you'll want to use `#[cfg]` instead of `if cfg!(...)` in this scenario to ensure the codegen is improved enough in debug mode that we can still handle it.

(same for the block below with `String`)

Reply:
Interesting, that seems like quite the limitation indeed.